compiler/ballerina-parser/src/test/resources/declarations/annot-decl/annot_decl_source_12.bal

```diff
@@ -0,0 +1,10 @@
+const annotation name on s //<cursor>
+
+Animal a = new Animal();
+
+const annotation name on source o //<cursor>
+
+Animal a = new Animal();
+
+const annotation name on source object f //<cursor>
+// Intentionally left with EOF as next token
```

Comment:
Should we add the other test case as well where the `object` contains a new line So that we can increase the coverage